pokemongo_bot/cell_workers/incubate_eggs_worker.py

```diff
@@ -2,65 +2,110 @@
 from pokemongo_bot.human_behaviour import sleep
```

Comment:
Please mind that I left sleep in just for the case that we need it lol. Guess I missed the line for distance. But I don't think the former worker as a ugly mess. There is no other way around to make it more beautiful, there is very much inventory parsing stuff in it. Since you didn't improve the code style so your code is basically a mess too ;)